src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,30 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.Table;
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
+@Table(name = "movies")
 public class Movie implements Cloneable {
```

Comment:
Implementing Cloneable and providing a clone method is not typical for entity classes in Hibernate. Entities are usually managed by the persistence context, and cloning can lead to unexpected behavior, especially with managed relationships and IDs. If you need to create a copy of an entity, consider using copy constructors or dedicated methods that clearly define what is being copied and how.